editoast/src/generated_data/mod.rs

```diff
@@ -86,37 +86,39 @@ pub trait GeneratedData {
 }
 
 /// Refresh all the generated data of a given infra
+#[tracing::instrument(level = "debug", skip_all, fields(infra_id))]
 pub async fn refresh_all(
-    db_pool: Arc<DbConnectionPool>,
-    infra: i64,
+    db_pool: Arc<DbConnectionPoolV2>,
```

Comment:
Likewise about the pool

Reply:
As we discussed, we will leave it as it is.